pwndbg/heap/ptmalloc.py

```diff
@@ -908,11 +907,14 @@ def is_glibc_static_symbol(self, addr):
         # Note: This might fail if there is an another symbol also call `_IO_list_all`...
         # TODO/FIXME: Find a better way to check if `addr` is a static symbol of GLIBC.
```

Comment:
btw what do we mean by 'addr being static symbol in libc'?

Reply:
That's because if some binaries have the same name symbols, I mean:

If we compiled this file with `gcc FILENAME.c -g` or `gcc FILENAME.c -static -g`:

```c
#include <stdlib.h>
void *main_arena = 0xdeadbeaf;
int main(){
    free(malloc(0x20));
    return 0;
}
```

`pwndbg.symbol.address("main_arena")` will return the address of this 0xdeadbeaf `main_arena` instead of the "real" `main_arena`, so the `heap` in debug symbols mode will fail.
But I want to avoid this and make it works in heuristic mode if possible, this why I have this weird check XD.

I'll check if `gdb.lookup_static_symbol` can help later.